pkg/images/integration/set.go

```diff
@@ -25,6 +26,14 @@ type Set interface {
 // NewSet returns a new Set instance.
 func NewSet(reporter integrationhealth.Reporter) Set {
 	registryFactory := registries.NewFactory(registries.FactoryOptions{})
+	if env.DisableRegistryRepoList.BooleanSetting() {
+		registryFactory = registries.NewFactory(registries.FactoryOptions{
```

Comment:
nit: I think we should define a variable for `CreatorFuncs` and then conditionally populate that instead of calling `NewFactory` twice

Reply:
Your suggestion is a better approach. This code no longer exists in based on changes made due to other code review comments.